examples/send_remote_finish.c

```diff
@@ -0,0 +1,30 @@
+#include <stdbool.h>
+#include <stdio.h>
+#include "../mimpi.h"
+#include "test.h"
+
+#define MESSAGE "Litwo ojczyzno moja Ty jestes jak zdrowie ile Cie trzeba cenic ten tylko sie dowie, kto Cię stracil..."
```

Comment:
Maybe use the exact quotation here, without any changes? ;)
Be kind for the original author.

Reply:
You mean leaving polish letters or have I made mistake (I was writing from my memory)?

Reply:
Original:
```
Litwo, Ojczyzno moja! ty jesteś jak zdrowie;
Ile cię trzeba cenić, ten tylko się dowie,
Kto cię stracił. 
```
you omitted exclamation mark, used Polish letters only in some places.

Reply:
Ok, I've corrected myself.